src/olympia/files/models.py

```diff
@@ -566,6 +573,16 @@ def add_file(self, chunks, filename, size):
         self.hash = 'sha256:%s' % hash_func.hexdigest()
         self.save()
 
+    def generate_access_token(self):
+        return binascii.hexlify(os.urandom(20)).decode()
```

Comment:
We now have pretty similar code like that in api.models for ApiToken, accounts.utils regarding fxa and others, I wonder if we could move that out. On the other hand, we won't gain much so it's rather a nit rather than a requirement.

Reply:
This would be a good follow-up after https://github.com/mozilla/addons-server/pull/12230 is also merged. I thought about it and then discarded the idea because the use cases were a little different, and ultimately it's just a one-liner, but maybe it's worth consolidating.

Reply:
Yeah, in particular that (not complex) one-linter part is what makes it only a nit. I wrote the comment down, wanted to deleted at first but then thought… meh… could be done, not a requirement though, so :man_shrugging: